### PR TITLE
Integration test fixs

### PR DESCRIPTION
**Pull request by vshanthe**
## 📝 Description

Below test cases are failing in Test Outcome Sum
- tests.integration.firewalls.test_firewalls_rules/test_list_rules_json_format
- tests.integration.regions.test_plugin_region_table/test_output

**What does this PR do and why is this change necessary?**
Fixed the failing test cases 

## ✔️ How to Test
**How do I run the relevant unit/integration tests?**

run all test 
make testint

To run individual module where fixed has been implemented 
make INTEGRATION_TEST_PATH=regions testint 
make INTEGRATION_TEST_PATH=firewalls/test_firewalls_rules.py testint
 

## 📷 Preview

**If applicable, include a screenshot or code snippet of this change. Otherwise, please remove this section.**

### CODE REVIEW

File: tests/integration/regions/test_plugin_region_table.py [hunk 1]
```diff
@@ -11,13 +12,16 @@ def exec_test_command(args: List[str]):
...
-        assert "-" in line
-        assert "✔" in line
-        assert "|" in line
+        check_marks = re.findall("✔", line)
+        hyphens = re.findall("-", line)
+        pipes = re.findall(r"\|", line)
+
+        assert all(mark in line for mark in check_marks)
+        assert all(hyphen in line for hyphen in hyphens)
+        assert all(pipe in line for pipe in pipes)
```
Comment (zliang-akamai) [inline tests/integration/regions/test_plugin_region_table.py:27]: Nice work! I wonder why we have to loop through `check_marks` if it’s all `✔`?

**Comment by zliang-akamai** [comment]
And you can try `make format` to fix the lint

File: tests/integration/regions/test_plugin_region_table.py (hunk 1 above)
Comment (jriddle-linode) [reply tests/integration/regions/test_plugin_region_table.py:27]: The lines are horizontal, i believe this is checking to see if there are checks, hyphens, and pipes on every line which is probably safe until we hit a datacenter capable of every possible feature.

**Review by jriddle-linode — APPROVED** [review]
Looks good to me.

**Review by zliang-akamai — APPROVED** [review]

File: tests/integration/regions/test_plugin_region_table.py (hunk 1 above)
Comment (zliang-akamai) [reply tests/integration/regions/test_plugin_region_table.py]: But it looks like we also to assert mark in the line if we found the mark in the line, which is kind of weird..

**Review by zliang-akamai — CHANGES_REQUESTED** [review]
I found the issue was caused by the program thinks the console columns is limited so it display some empty lines, like:
image

I think we can use the COLUMNS env var to fix this issue. I found this is a good example for it:
```python
import subprocess
import os

# Set the console width to 100
env = os.environ.copy()
env["COLUMNS"] = "100"

process = subprocess.run(
    args,
    stdout=subprocess.PIPE,
    env=env,
)

```

File: tests/integration/regions/test_plugin_region_table.py (hunk 1 above)
Comment (vshanthe) [reply tests/integration/regions/test_plugin_region_table.py]: It was taking the column outer line as well for the asserts which was like this - ┡━━━━━━━╇━━━━━━━╇━━━━━╇━━━━━━╇━━━━━╇━━━━╇━━━━━╇━━━━╇━━━━╇━━━━━╇━━━━━━╇━━━━━━━╇━━━━━━╇━━━━━━╇━━━━━━━┩'

**Review by vshanthe — COMMENTED** [review]
Changes implemented

**Comment by zliang-akamai** [comment]
It's passing now, great work!
https://github.com/linode/linode-cli/actions/runs/6742822588

**Review by zliang-akamai — APPROVED** [review]

**Pull request merged**